akka-persistence/src/main/scala/akka/persistence/journal/AsyncWriteJournal.scala

```diff
@@ -55,109 +54,114 @@ trait AsyncWriteJournal extends Actor with WriteJournalBase with AsyncRecovery {
   final def receive = receiveWriteJournal.orElse[Any, Unit](receivePluginInternal)
 
   final val receiveWriteJournal: Actor.Receive = {
-    case WriteMessages(messages, persistentActor, actorInstanceId) ⇒
-      val cctr = resequencerCounter
-      resequencerCounter += messages.foldLeft(0)((acc, m) ⇒ acc + m.size) + 1
+    // cannot be a val in the trait due to binary compatibility
+    val replayDebugEnabled: Boolean = config.getBoolean("replay-filter.debug")
```

Comment:
This is the only change, the rest is just indentation (check [here](https://github.com/akka/akka/pull/19644/files?w=1)).